src/tools/rbd_mirror/ImageReplayer.cc

```diff
@@ -149,6 +150,25 @@ class FlushCommand : public ImageReplayerAdminSocketCommand {
 };
 
 template <typename I>
+class ResyncCommand : public ImageReplayerAdminSocketCommand {
```

Comment:
I thought you were going to yank the ASOK resync command since you can just use the rbd CLI?